simulation/app.py

```diff
@@ -0,0 +1,621 @@
+                            ]),
+                        ]),
+                        html.Tr([
+                            html.Td([
+                                html.Label('Move length', htmlFor='move-length'),
+                                dcc.Slider(id='move-length',
+                                           min=0,
+                                           max=1,
+                                           step=0.1,
+                                           value=0.5,
+                                           marks={
+                                               0: {'label': '0'},
+                                               1: {'label': '1'}
+                                           })
+                            ]),
+                            html.Td([
+                                html.P('0', id='move-dist-value')
+                            ]),
+                        ])
+                    ], style={'width': '468px'}),
+                    html.Div([
+                        html.Button('Start', id='start-button', n_clicks=0),
+                        html.Button('Stop', id='stop-button', n_clicks=0),
+                        html.Button('Continue', id='continue-button', n_clicks=0),
+                        html.Button('Reset', id='reset-button', n_clicks=0),
+                    ], style={'display': 'flex', 'justify-content': 'center', 'padding': '5px'})
+
+            ])
+        ], style={'display': 'flex', 'justify-content': 'space-evenly'}),
+        html.Div([
+            html.Div([
+                html.Table(id='current-population-amount', style={
+                })
+            ]),
+            html.Div(
+                html.Table(id='additional-parameters')),
+        ], style={'display': 'flex', 'justify-content': 'space-evenly'}),
+
+
+
+        html.Div(
+            dcc.Graph(id='live-population'), style={'marginTop': '20px'}),
+        html.Div([
+            dcc.Graph(id='move-population'),
+            dcc.Graph(id='population-percent'),
+        ], style={'display': 'flex', 'justify-content': 'space-evenly', 'marginTop': '20px'}),
+        dcc.Store(id='simulation-data')
+    ])
+
+
+@app.callback(Output('inf-prob-value', 'children'),
+              [Input('infection-probability', 'value')])
+def update_slider(value):
+    return value
+
+
+@app.callback(Output('rec-prob-value', 'children'),
+              [Input('recover-probability', 'value')])
+def update_slider(value):
+    return value
+
+
+@app.callback(Output('dead-prob-value', 'children'),
+              [Input('dead-probability', 'value')])
+def update_slider(value):
+    return value
+
+
+@app.callback(Output('inf-range-value', 'children'),
+              [Input('infection-range', 'value')])
+def update_slider(value):
+    return value
+
+
+@app.callback(Output('move-dist-value', 'children'),
+              [Input('move-length', 'value')])
+def update_slider(value):
+    return value
```

Comment:
Here you have redeclaration of functions even without description. Ofc. it is better here to name function individually than putting docstrings as these functions are really small.